R/bin_time.R

```diff
@@ -302,7 +304,7 @@ bin_time <- function(occdf, bins, method = "mid", reps = 100,
         occdf$bin_assignment[id_vec] <- bin_list[[i]]
       }
       # Add bin midpoints to dataframe
-      for (i in seq_len(nrow(occdf))) {
+      for (i in seq_len(nrow(bins))) {
```

Comment:
Any particular reason this has been changed? Sorry, can't figure it out immediately.

Reply:
This seems to have been a bug? Previously we were using the row number of the `occdf` to index through bins, but shouldn't it be the bin?

Reply:
Yes, you're correct; done a few tests and changing this increases speed by a large amount. Nice catch!

Reply:
What terrible reviewer let that bug through?! /s

Reply:
Unfortunately, we're all human 😅